Updates to process_mon.sh and ubuntu_updates_check.sh

tcaddy opened the pull request:
## Updates to process_mon.sh
Add an additional metric `process_age` to the `process_mon.sh` script.  This will allow you to raise an alarm if a process has been running for too long.

## Updates to ubuntu_update_check.sh
* `/usr/lib/update-notifier/apt-check` was not returning a count of upgradeable packages on various machines even though you could see packages available for update via `apt-get upgrade`
 * Replace with `apt list --upgradeable` and parse out the count of upgradeable security and non-security package updates available

tcaddy committed:
Get process age metric

itzg commented on process_mon.sh:
```diff
@@ -64,4 +68,12 @@ if [ -z "$1" ]; then
 
 process_check=`ps ax | grep -v grep | grep -v process_mon | grep -c "$1"`
+if (( $process_check > 0 )); then
+  process_age=`ps -o etimes= -p "$process_check"`
```
Since `$process_check` will contain either 0 or 1 from the `grep -c` this doesn't seem to grab the `etimes` from the intended pid.

tcaddy commented:
Oops thought it was getting a PID.  Let me try again

tcaddy committed:
Update ubuntu_update_check.sh

 * /usr/lib/update-notifier/apt-check was not returning a count of
   upgradeable packages on various machines even though you could
   see packages available for update via `apt-get update`
 * Replace with `apt list --upgradeable` and parse out the count
   of upgradeable security and non-security package updates available

itzg commented on ubuntu_updates_check.sh:
```diff
@@ -23,15 +23,15 @@
-PENDING_OTHER=$(echo "${OUTPUT}" | cut -d ";" -f 1)
-PENDING_SECURITY=$(echo "${OUTPUT}" | cut -d ";" -f 2)
+PENDING_OTHER=$(echo "${OUTPUT}" | grep -v "Listing..." | grep -v "/$(lsb_release -cs)-security" | wc -l)
+PENDING_SECURITY=$(echo "${OUTPUT}" | grep -v "Listing..." | grep "/$(lsb_release -cs)-security" | wc -l)
```
I did a quick test and noticed that some packages on my system are available via both channels, so the pattern here needs to match a leading slash or comma in order to match:

```
ruby1.9.1-dev/trusty-updates,trusty-security 1.9.3.484-2ubuntu1.13 amd64 [upgradable from: 1.9.3.484-2ubuntu1.7]
```

tcaddy committed:
Add `file_info.sh`

tcaddy committed:
Get port_check.sh to work on Ubuntu 18.04 LTS

tcaddy committed:
Ignore held packages in `ubuntu_updates_check.sh`

tcaddy committed:
add `ssl_verify.sh`

tcaddy committed:
Add `ssl_ca_expiration_check.sh`

tcaddy committed:
Make `cloudbackup_mon.sh` work with python3

tcaddy committed:
make file_info.py work with python3

tcaddy committed:
Specify `python2` in `holland_mysqldump.py` so it will work
